site/src/xServices/updateCheck/updateCheckXService.ts

```diff
@@ -0,0 +1,172 @@
+import { assign, createMachine } from "xstate"
+import { checkAuthorization, getUpdateCheck } from "api/api"
+import { AuthorizationResponse, UpdateCheckResponse } from "api/typesGenerated"
+
+export const checks = {
+  viewUpdateCheck: "viewUpdateCheck",
+} as const
+
+export const permissionsToCheck = {
+  [checks.viewUpdateCheck]: {
+    object: {
+      resource_type: "update_check",
+    },
+    action: "read",
+  },
+} as const
```

Comment:
I'm not sure why we need to export these constants or why we need to cast as `const` although I see this pattern in our other machines. @presley (or @mafredri) is there something dumb I'm missing?

Reply:
Same here, I thought it was weird but I kept them just in case. I'll remove them and see if something breaks 👍🏻